src/components/SourcesFilter.js

```diff
@@ -11,7 +11,7 @@ class SourcesFilter extends Component {
         <SimpleTableFilter
             buttonTitle={null}
             placeholder='Filter by source name'
-            options={{
+            xoptions={{
```

Comment:
Is this supposed to be `xoptions`?

Reply:
Yes, I am disabling it (temporary) by renaming the option.